dso-l2/src/main/java/com/tc/l2/state/StateManagerImpl.java

```diff
@@ -210,7 +210,12 @@ private synchronized void moveToPassiveReady(Enrollment winningEnrollment) {
     logger.debug("moving to passive ready " + state + " " + winningEnrollment);
     if (state == START_STATE) {
       setActiveNodeID(winningEnrollment.getNodeID());
-      state = PASSIVE_UNINITIALIZED;
+      state = (startState == null) ? PASSIVE_UNINITIALIZED : startState;
+      if (!state.equals(PASSIVE_STANDBY) && !state.equals(PASSIVE_UNINITIALIZED)) {
+// TODO:  make sure this is the proper way to handle this.
```

Comment:
Don't we need to re-sync a server with Active if it went down as a PASSIVE_STANDBY?